frontend/server/src/main/java/com/amazonaws/ml/mms/util/ConfigManager.java

```diff
@@ -550,13 +570,24 @@ public static Options getOptions() {
                             .argName("MODELS-STORE")
                             .desc("Model store location where models can be loaded.")
                             .build());
+            options.addOption(
```

Comment:
Not necessary to make it a cli option

Reply:
I had similar reasoning here as the Environment variable. This keeps MMS config consistent.